spring-cloud-starter-task-composedtaskrunner/pom.xml

```diff
@@ -14,8 +14,8 @@
     </parent>
 
     <properties>
-        <spring-cloud-common-security-config.version>1.2.0.BUILD-SNAPSHOT</spring-cloud-common-security-config.version>
```

Comment:
Not sure I understand this. Could you explain the reason not to use the `1.2.0.BUILD-SNAPSHOT` of common security config?

Reply:
Learned that we can't use Spring Boot 2.2 yet and that's the reason we can't upgrade to this version.